src/models/genres.js

```diff
@@ -0,0 +1,24 @@
+class Genres {
+}
+
+Genres.fields = ['acoustic', 'afrobeat', 'alt-rock', 'alternative', 'ambient',
+                 'anime', 'black-metal', 'bluegrass', 'blues', 'bossanova', 'brazil',
+                 'breakbeat', 'british', 'cantopop', 'chicago-house', 'children', 'chill',
+                 'classical', 'club', 'comedy', 'country', 'dance', 'dancehall', 'death-metal',
+                 'deep-house', 'detroit-techno', 'disco', 'disney', 'drum-and-bass', 'dub',
+                 'dubstep', 'edm', 'electro', 'electronic', 'emo', 'folk', 'forro', 'french',
+                 'funk', 'garage', 'german', 'gospel', 'goth', 'grindcore', 'groove', 'grunge',
+                 'guitar', 'happy', 'hard-rock', 'hardcore', 'hardstyle', 'heavy-metal', 'hip-hop',
+                 'holidays', 'honky-tonk', 'house', 'idm', 'indian', 'indie', 'indie-pop',
+                 'industrial', 'iranian', 'j-dance', 'j-idol', 'j-pop', 'j-rock', 'jazz', 'k-pop',
+                 'kids', 'latin', 'latino', 'malay', 'mandopop', 'metal', 'metal-misc', 'metalcore',
+                 'minimal-techno', 'movies', 'mpb', 'new-age', 'new-release', 'opera', 'pagode',
+                 'party', 'philippines-opm', 'piano', 'pop', 'pop-film', 'post-dubstep', 'power-pop',
+                 'progressive-house', 'psych-rock', 'punk', 'punk-rock', 'r-n-b', 'rainy-day',
+                 'reggae', 'reggaeton', 'road-trip', 'rock', 'rock-n-roll', 'rockabilly', 'romance',
+                 'sad', 'salsa', 'samba', 'sertanejo', 'show-tunes', 'singer-songwriter', 'ska',
+                 'sleep', 'songwriter', 'soul', 'soundtracks', 'spanish', 'study', 'summer',
+                 'swedish', 'synth-pop', 'tango', 'techno', 'trance', 'trip-hop', 'turkish',
+                 'work-out', 'world-music']
```

Comment:
I wonder if it would be better to get the list of genres via a Spotify API call, in case they add/remove some in the future. Was the list available via an endpoint?